packages/playground/src/dashboard/components/NodeMintingDetails.vue

```diff
@@ -1,65 +1,94 @@
-              <v-select v-model="selectedMonth" :items="months" label="Month"></v-select>
-
-              <v-select v-model="selectedYear" :items="years" label="Year"></v-select>
-            </v-card-text>
-          </v-card>
-        </v-menu>
-      </v-col>
-      <v-col cols="9">
-        <v-card outlined>
-          <v-card-title>
-            <span class="headline">Minting Details</span>
-          </v-card-title>
-          <div class="receipt-body" v-if="receipts?.length">
-            <v-card outlined v-for="receipt in receipts" :key="receipt.hash">
+      <v-card>
+        <v-card-text class="d-flex">
+          <v-select v-model="selectedMonth" :items="months" label="Month"></v-select>
+          <v-select v-model="selectedYear" :items="years" label="Year"></v-select>
+        </v-card-text>
+      </v-card>
+    </v-menu>
+    <v-card outlined class="pa-2 border">
+      <div class="receipt-body" v-if="receipts?.length">
+        <v-row>
+          <v-col
+            class="text-start"
+            cols="12"
+            md="12"
+            :lg="receipts.length > 1 ? '6' : '12'"
+            v-for="receipt in receipts"
+            :key="receipt.hash"
+          >
+            <v-card style="box-shadow: none" outlined>
               <v-container>
                 <v-row>
-                  <v-col cols="10">
-                    <v-card-text>
-                      <span class="font-weight-bold">Date of Payout :</span>
-                      {{ getDateFromTimestamp(receipt.endPeriodTimestamp) }}
-                      <br />
-                      <span class="font-weight-bold">Node TFT Amount :</span> {{ receipt.tft || 0 }} TFT
-                      <span v-if="receipt.fixupReward">+ {{ receipt.fixupReward || 0 }} TFT FixedUp</span>
-                      <br />
-                      <span class="font-weight-bold">Cloud Units :</span>
-                      <ul>
-                        <li inset v-for="(val, key) in receipt.cloud_units" :key="key">
-                          <b style="text-transform: uppercase">{{ key }}:</b> {{ val }}
-                        </li>
-                      </ul>
-                    </v-card-text>
-                  </v-col>
-                  <v-col cols="2">
-                    <v-chip class="ma-4 me-auto" small :color="getChipColor(receipt.type)">{{ receipt.type }} </v-chip>
-                  </v-col>
+                  <v-list class="custom-list" density="compact">
+                    <h3 class="mb-1 font-weight-medium">
+                      Minting Details<v-chip class="ma-2 me-auto" small :color="getChipColor(receipt.type)"
+                        >{{ receipt.type }}
+                      </v-chip>
+                    </h3>
+                    <div class="border">
+                      <v-row class="row-style">
+                        <v-col class="py-1" cols="1" sm="2" style="min-width: fit-content">
+                          <v-list-item> Date of Payout :</v-list-item>
```

Comment:
looks good just please make this justify-content end to be consistent with other fields